<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaTraining</class>
 <widget class="QWidget" name="MediaTraining">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>621</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>-1</number>
    </property>
    <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
     <widget class="QLabel" name="questionText">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: green;
    color: white;
    text-align: left;
    padding: 4px;  
    border: none;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="answerWidget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="option2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: green;
    color: white;
    text-align: left;
    padding: 4px;  
    border: none;
}
QPushButton:hover {
    background-color: #222222;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="option4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: green;
    color: white;
    text-align: left;
    padding: 4px;  
    border: none;
}
QPushButton:hover {
    background-color: #222222;
}
</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="option3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: green;
    color: white;
    text-align: left;
    padding: 4px;  
    border: none;
}
QPushButton:hover {
    background-color: #222222;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="option1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: green;
    color: white;
    text-align: left;
    padding: 4px;  
    border: none;
}
QPushButton:hover {
    background-color: #222222;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="pressConferenceImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>661</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="contentBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 1px solid #000000;
    border-radius: 4px;
    background-color: #333333; 
    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: #00aa00; 
    width: 5px; 
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <zorder>pressConferenceImage</zorder>
  <zorder>widget</zorder>
  <zorder>contentBar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
